tools/deploy.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e
+
+cd ../cypher-codemirror
+
+yarn run build-demo
+
+cd demo
+
+git init
+
+git remote add origin git@github.com:neo4j-contrib/cypher-editor.git
+
+git checkout -b gh-pages
+
+git add --all
+
+git commit -m "Deploy website"
+
+git push origin gh-pages
```

Comment:
I believe that this script will be successful only once.

After first run we will have 1 commit in gh-pages branch. Therefore on trying to push there again will lead to reject from server, because we already have commit in a branch.